mix/qml/Debugger.qml

```diff
@@ -219,6 +219,33 @@ Rectangle {
 							anchors.horizontalCenter: parent.horizontalCenter
 							id: jumpButtons
 							spacing: 3
+
+                            StepActionImage
```

Comment:
We use tab for indentation, you can configure Qt to use indentation there:
Tools => Option => Tabs And Indentation => TabsOnly (TabSize 4 && IndentSize 4)